protocols/piedao/events.json

```diff
@@ -5,4 +5,10 @@
     "date": "2021-10-08T06:00:00.000Z",
     "type": "call"
   }
```

Comment:
you need a comma here

Reply:
Thanks, now I've learned that for proper syntax events must be delimited by a comma.